### ISSUE TITLE
Borderlands 3 - Fails downloads

### ISSUE THREAD

**Issue by tpill90** [tpill90/epic-lancache-prefill#8]
![image](https://user-images.githubusercontent.com/766513/218349484-1f49be2a-a5db-43f5-baa7-174b5abb7608.png)

Likely related to DLC

**Comment by itapai** [comment]
When downloading Fortnite or Fall Guys, i get the same error, so I don't think it related to DLC.

When I download the game from the Epic Launcher, I get no errors in the lancache logs.
When I download with epic-lancache-prefill, the error logs are full of lines like this:
```
2023/02/16 10:38:12 [error] 1819#1819: *1581 unexpected status code 200 in slice response while reading response header from upstream, client: 172.19.0.1, server: , request: "GET /Builds/Fortnite/CloudDir/ChunksV4/67/D4B0A86DBE55CD64_AD8E304E474999D827605FB33D1D8F8C.chunk HTTP/1.1", subrequest: "/Builds/Fortnite/CloudDir/ChunksV4/67/D4B0A86DBE55CD64_AD8E304E474999D827605FB33D1D8F8C.chunk", upstream: "http://127.0.0.1:3128/Builds/Fortnite/CloudDir/ChunksV4/67/D4B0A86DBE55CD64_AD8E304E474999D827605FB33D1D8F8C.chunk", host: "fastly-download.epicgames.com"
2023/02/16 10:38:12 [error] 1816#1816: *1619 unexpected status code 200 in slice response while reading response header from upstream, client: 172.19.0.1, server: , request: "GET /Builds/Fortnite/CloudDir/ChunksV4/97/5AE8A10FDB40C0FC_BF6F39674E003D34C31A65A56F15C380.chunk HTTP/1.1", subrequest: "/Builds/Fortnite/CloudDir/ChunksV4/97/5AE8A10FDB40C0FC_BF6F39674E003D34C31A65A56F15C380.chunk", upstream: "http://127.0.0.1:3128/Builds/Fortnite/CloudDir/ChunksV4/97/5AE8A10FDB40C0FC_BF6F39674E003D34C31A65A56F15C380.chunk", host: "fastly-download.epicgames.com"
2023/02/16 10:38:12 [error] 1818#1818: *1622 unexpected status code 200 in slice response while reading response header from upstream, client: 172.19.0.1, server: , request: "GET /Builds/Fortnite/CloudDir/ChunksV4/62/5113E39726A9FCEC_D4D531C94093083997CEE2A4023036D1.chunk HTTP/1.1", subrequest: "/Builds/Fortnite/CloudDir/ChunksV4/62/5113E39726A9FCEC_D4D531C94093083997CEE2A4023036D1.chunk", upstream: "http://127.0.0.1:3128/Builds/Fortnite/CloudDir/ChunksV4/62/5113E39726A9FCEC_D4D531C94093083997CEE2A4023036D1.chunk", host: "fastly-download.epicgames.com"
2023/02/16 10:38:12 [error] 1817#1817: *1603 unexpected status code 200 in slice response while reading response header from upstream, client: 172.19.0.1, server: , request: "GET /Builds/Fortnite/CloudDir/ChunksV4/74/F9549A11F9D39F1C_ED8F4905447448246269B88EB5CC091B.chunk HTTP/1.1", subrequest: "/Builds/Fortnite/CloudDir/ChunksV4/74/F9549A11F9D39F1C_ED8F4905447448246269B88EB5CC091B.chunk", upstream: "http://127.0.0.1:3128/Builds/Fortnite/CloudDir/ChunksV4/74/F9549A11F9D39F1C_ED8F4905447448246269B88EB5CC091B.chunk", host: "fastly-download.epicgames.com"
```

**Comment by tpill90** [comment]
Released in [v1.1.0](https://github.com/tpill90/epic-lancache-prefill/releases/tag/v1.1.0)
